lib/clientv2/globalnetworkpolicy.go

```diff
@@ -16,6 +16,7 @@ package clientv2
 
```

Comment:
GlobalNetworkPolicies don't need any of this special name handling since we don't have any k8s based policies show up here.

Reply:
Apologies - I thought I'd written that in one of my comments and it looks like i did not.

Reply:
Oh actually, yes I did.

Reply:
Oh oops missed that one. My bad.

Reply:
From our conversation about this, I will be leaving the prefixing logic in the GlobalNetworkPolicy as well since we need it for non k8s based policies as well.

Reply:
Yup - my bad.